view/frontend/templates/promo/product-promotional-messaging.phtml

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Amazon\Pay\Block\Promo\PromoMessaging;
+use Magento\Framework\Escaper;
+
+/**
+ * @var PromoMessaging $block
+ * @var Escaper $escaper
+ */
+
+$isEligibleProduct = $block->checkIsEligibleProduct();
+?>
+
+<?php if ($isEligibleProduct): ?>
+
+    <div class="ap-promotional-message"></div>
```

Comment:
I worry about hard-coding a class here and also in the js — Would it be better to use the `x-magento-init` to target the element or pass in the target as a config option?

Reply:
yeah i like that!